### ISSUE TITLE
supported os for hs2client

### ISSUE THREAD

**Issue by sanghkaang** [cloudera/hs2client#22]
Hello.

I want to know supported os for hs2client.
I tried to install hs2client on my mac but i failed.

Does hs2client supports mac OS?

If not, for newbie like me, how about specify `supported os for hs2client` at readme.md?

**Comment by wesm** [comment]
Yes, it does support macOS. I am not able to support the library right now, but if you show the build error, others may be able to help.
